app/views/content/non-uk-teachers/non-uk-qualifications.md

```diff
@@ -130,4 +128,8 @@ If you're already a qualified teacher, find out how you can [teach in England if
 
 If you have general questions about the information on this page, you can email us at teach.inengland@education.gov.uk.
 
-For help with individual applications to training providers or schools, please [register with a teacher training adviser](/teacher-training-adviser/sign_up/identity). Advisers can give you personalised support with your application.
+You may be eligible for personalised support with your application to a training provider or school from a [teacher training adviser](/teacher-training-advisers).
```

Comment:
@laurahermionetennant92 same as previous comment I found this sentence a bit confusing. Perhaps it might be easier to split into 2 sentences?

You may be eligible for personalised support from a [teacher training adviser](https://github.com/teacher-training-advisers). They can help with your application to a training provider or school.